MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -3330,6 +3341,7 @@
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
+				STRIP_SWIFT_SYMBOLS = YES;
```

Comment:
Should we be taking this change, and other adjacent project file changes related to symbol stripping? It seems they should maybe go into a separate PR if they're intentional.

Reply:
Good catch. Reverted.